### DISCUSSION TITLE
数组容器可以变更顺序吗？

### DISCUSSION THREAD

**Discussion by Channing-Huang** [jd-opensource/drip-form#170] [Q&A]
数组容器目前能否扩展拖拽变更顺序？或者后续会支持吗？

**Comment by mengshang918** [comment]
目前不支持，后续会支持，已在规划中。

**Reply by Channing-Huang** [reply]
方便问下大概在哪个版本会支持吗？或者有roadMap吗？

**Reply by mengshang918** [reply]
v0.8.0。roadmap目前是团队的，后续迁移下

**Reply by mengshang918** [reply]
> 方便问下大概在哪个版本会支持吗？或者有roadMap吗？

readme已经添加roadmap。可以查看readme，或者直接查看https://github.com/orgs/JDFED/projects/2/views/7
